packages/app/src/components/Dimensions/RecommendedIcon.js

```diff
@@ -39,7 +35,7 @@ export class RecommendedIcon extends Component {
             ? this.showTooltip()
             : null;
 
-        return this.checkIfRecommended() ? (
+        return Boolean(this.props.isRecommended) ? (
```

Comment:
Is it necessary to cast to Boolean?  isRecommended comes from javascript array.includes, which returns a boolean already